docs/source/x_ext.rst

```diff
@@ -531,13 +503,83 @@ The |processor| shall cause an illegal instruction fault when attempting to exec
 A |coprocessor| can (only) accept an offloaded instruction when:
 
 * It can handle the instruction (based on decoding ``instr``).
-* The required source registers are marked valid by the offloading core  (``issue_valid`` is 1 and required bit(s) ``rs_valid`` are 1).
+* There are no structural hazards that would prevent execution
```

Comment:
I would remove this bullet. If not, then it needs to be explained what structural hazards a coprocessor is supposed to check (as most of these checks should actually be performed by the processor, not the coprocessor).

Reply:
I would agree that this bullet is somewhat redundant to the first, but it is more specific. What I am trying to state is that the coprocessor would have to check if the necessary execution units are available. Thinking further, as coprocessors may implement custom registers, there may even be data hazards only visible in the coprocessor. Maybe a good topic for discussion

Reply:
I think those are all details that only matter to the coprocessor itself and are confusing to address in the XIF spec. Of course both the processor and coprocessor needs to deal with potential hazards, but we are not going to list them here.

Reply:
Implementation hints can absolutely be useful to understand the specification more quickly.